skimage/draw/_random_shapes.py

```diff
@@ -171,46 +171,50 @@ def _generate_triangle_mask(point, image, shape, random):
 SHAPE_CHOICES = list(SHAPE_GENERATORS.values())
 
 
-def _generate_random_color(gray, min_pixel_intensity, random):
+def _generate_random_color(num_channels, intensity_range, random):
```

Comment:
I feel like this should be vectorised: `_generate_random_colors(num_colors, num_channels, intensity_range, random)`, returning a `(num_colors, num_channels)` shaped-array.